themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -136,6 +136,7 @@
                 pattern="[0-9]*"
                 value="{$product.quantity}"
                 name="product-quantity-spin"
+                aria-label="{$product.name} {l s='product quantity field' d='Shop.Theme.Checkout'}"
```

Comment:
Will this wording be visible on the checkout page? If yes, where will it appear exactly? I thought this PR didn't have any visible change so I don't understand why there is a new wording. Thank you in advance. :)

Reply:
This wording will be spoken by a screen reader

Reply:
Thanks!

Reply:
Don't hesitate to tell me if the wording is wrong @Julievrz, as it will be spoken, we can considere it as a shown string!

Reply:
Ok, I'll have a look now, thank you!